apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QIntegrationTestBase.java

```diff
@@ -288,33 +287,30 @@ protected void createKeyspace(String keyspaceName) {
    */
 
   protected NameResponse createSimpleTestTable(String keyspaceName, String tableName) {
-    final Sgv2TableAddRequest tableAdd = new Sgv2TableAddRequest(tableName);
-    tableAdd.setColumnDefinitions(
+    List<Sgv2ColumnDefinition> columnDefs =
         Arrays.asList(
             new Sgv2ColumnDefinition("id", "uuid", false),
             new Sgv2ColumnDefinition("lastName", "text", false),
             new Sgv2ColumnDefinition("firstName", "text", false),
-            new Sgv2ColumnDefinition("age", "int", false)));
-
-    Sgv2Table.PrimaryKey primaryKey = new Sgv2Table.PrimaryKey();
-    primaryKey.setPartitionKey(Arrays.asList("id"));
-    tableAdd.setPrimaryKey(primaryKey);
+            new Sgv2ColumnDefinition("age", "int", false));
 
+    Sgv2Table.PrimaryKey primaryKey = new Sgv2Table.PrimaryKey(Arrays.asList("id"), null);
```

Comment:
nit: maybe add constructor that only accepts the list?

Reply:
Added.